dal/dal.go

```diff
@@ -32,6 +31,14 @@ type IDal interface {
 	FetchAlerterConfig(string) (string, error)
 }
 
+type GetOptions struct {
+	Recurse  bool
+	NoPrefix bool
```

Comment:
This option allows us to tell the dal whether to attach a prefix (ie. `9volt`) or not. This is useful if a particular dal method is using the dal itself (and wants to avoid double-adding the prefix).